qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -464,6 +464,8 @@ def get_channel(self, channel: str) -> channels.PulseChannel:
                 return channels.MeasureChannel(index)
             elif prefix == channels.ControlChannel.prefix:
                 return channels.ControlChannel(index)
+            elif prefix == channels.MemorySlot.prefix:
```

Comment:
No, I'm not building custom backend model - I just picked this up because I wanted to get involved!

Just drawing from the issue and writing the following for documentation, as you're very likely to know this, at the moment it can be reached because if you call 

```python
from qiskit.pulse import Schedule, Play, Acquire, MemorySlot, AcquireChannel, DriveChannel
from qiskit.pulse.library.parametric_pulses import Constant
from qiskit.pulse.transforms import pad
from qiskit.qobj.converters.pulse_instruction import QobjToInstructionConverter
from qiskit import assemble

sched=Schedule()
sched+=Play(Constant(4, 0.1), DriveChannel(0))
sched+=Acquire(3, AcquireChannel(0), MemorySlot(0))
pad(sched, inplace=False)
``` 
the pad() line does this:

```python
   until = until or schedule.duration
    channels = channels or schedule.channels

    for channel in channels:
        if channel not in schedule.channels:
            schedule |= instructions.Delay(until, channel)
            continue
```
which creates a Delay on the MemorySlot 'channel'. 

However, as @wshanks has said in https://github.com/Qiskit/qiskit-terra/issues/7078#issuecomment-1155579469 this would be fixed if the Aquire.channels() did not return MemorySlots so it sounds like that's what makes sense, and doesn't require a new prefix.

Reply:
Since you added branching for memory slots in pad function, this is no longer necessary?